crossdock/server/endtoend.py

```diff
@@ -101,10 +106,19 @@ def tracers(self, tracers):
 
     @property
     def local_agent_sender(self):
+        try:
+            host, port_str = os.getenv('AGENT_HOST_PORT').split(':')
```

Comment:
this is rather dirty. If there's a legitimate error, like port not a number, it should be logged. But the way this try/catch is structured it can blow even when the env var is empty.

Reply:
Ya I don't really like it but this config module does stuff like this all over the place. I'll see what I can do.